pyart/correct/_fast_edge_finder.pyx

```diff
@@ -64,7 +64,8 @@ def _fast_edge_finder(
                             break
 
                 # add the edge to the collection (if valid)
-                collector.add_edge(label, neighbor, vel)
+                nvel = data[x_check, y_index]
```

Comment:
This will cause a segmentation fault when x_check is invalid.  Need to add a check before performing indexing.